### PR TITLE
Logistic Kit

### PR DESCRIPTION
**Pull request by KatherineMuedas**
#### Runway Ticket URL
#174 

#### Breaking Changes

_Please reflect if your changes may break in some way (changes like removing/renaming props are examples of breaking changes)_

#### Checklist:

- [x] **SPECS** Please cover your changes with specs
- [x] **CHANGELOG** Please add an entry on `[Unreleased]` section to every functionality `ADDED`/`CHANGED`/`DEPRECATED`/`REMOVED`/`FIXED`

### CODE REVIEW

**Comment by KatherineMuedas** [comment]
So far project kit is looking like this
Screen Shot 2020-03-19 at 12 09 00 PM

**Comment by KatherineMuedas** [comment]
Rails Project Kit
Screen Shot 2020-03-20 at 12 05 23 PM
Screen Shot 2020-03-20 at 2 10 50 PM

File: app/pb_kits/playbook/_playbook.scss [hunk 1]
```diff
@@ -66,3 +66,4 @@
 @import 'pb_user/user';
 @import 'pb_user_badge/user_badge';
 @import 'pb_filter/filter';
+@import 'pb_project/project';
```
Comment (jasoncypret) [inline app/pb_kits/playbook/_playbook.scss]: Can you put this in alphabetical order?

File: app/pb_kits/playbook/data/menu.yml [hunk 2]
```diff
@@ -74,3 +74,4 @@ kits:
     - title_detail
   - user
   - user_badge
+  - project
```
Comment (jasoncypret) [inline app/pb_kits/playbook/data/menu.yml]: Can you put this in alphabetical order + add it to the type patterns section?

File: app/pb_kits/playbook/packs/kits/pb_project.js [hunk 3]
```diff
@@ -0,0 +1 @@
+import '../../pb_project/_project.jsx'
```
Comment (jasoncypret) [inline app/pb_kits/playbook/packs/kits/pb_project.js]: This file is not used. We need to update the generator. Can you delete it? CC: @thestephenmarshall

File: app/pb_kits/playbook/pb_project/_project.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
+[class^=pb_project_kit] {
+  .links{
+    span{
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_project/_project.scss]: Can you make `links` selector more unique? Perhaps `pb_project_kit_links`? Can you add a space between the selector and the curly brace?

File: app/pb_kits/playbook/_playbook.scss (hunk 1 above)
Comment (KatherineMuedas) [reply app/pb_kits/playbook/_playbook.scss]: Sure! Changes here 2a431c6

File: app/pb_kits/playbook/data/menu.yml (hunk 2 above)
Comment (KatherineMuedas) [reply app/pb_kits/playbook/data/menu.yml]: This is alphabetical now. I am not sure what is `type patterns` but I think this is what you mean. Let me know if this is the wrong place :)
Changes here 2a431c6

File: app/pb_kits/playbook/packs/kits/pb_project.js (hunk 3 above)
Comment (KatherineMuedas) [reply app/pb_kits/playbook/packs/kits/pb_project.js]: The file has been removed.
Changes here 2a431c6

File: app/pb_kits/playbook/pb_project/_project.scss (hunk 4 above)
Comment (KatherineMuedas) [reply app/pb_kits/playbook/pb_project/_project.scss]: Good catch! Thanks!
Changes here 2a431c6

File: app/pb_kits/playbook/pb_project/_project.jsx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+  return ` · ${month}/${day}`
+}
+
+type ProjectProps = {
+  dark?: Boolean,
+  date: String,
+  link?: String,
+  projectName: String,
+  projectNumber: Number,
+}
```
Comment (KatherineMuedas) [inline app/pb_kits/playbook/pb_project/_project.jsx]: @jasoncypret @thestephenmarshall, I am wondering something, should I add to type ProjectProps 
  ```
  className?: String,
  data?: String,
  id?: String,
```

**Comment by jasoncypret** [comment]
@KatherineMuedas after chatting with the team, we need to make some slight adjustments to the name of the Kit. @jasperfurniss will be pushing up some changes shortly. Still hoping for this next release. Just keeping you in the loop.

**Pull request merged**